README.md

```diff
@@ -25,28 +28,38 @@ zplug "greymd/ttcopy"
 
 ## zsh, bash
 
+#### 1. Clone Repository
+
+```sh
+$ git clone https://github.com/greymd/ttcopy.git ~/ttcopy
+```
+
+#### 2. Edit `.bashrc` or `.zshrc`
+
+And add following lines.
+
 ```sh
 # Set ID and PASSWORD **AS YOU LIKE**.
```

Comment:
thx. I'll work on it.
README.md should be more updated (not only the part you pointed out).